doozer/doozerlib/cli/scan_sources_konflux.py

```
@@ -63,9 +64,10 @@ def __init__(
             raise DoozerFatalError('Cannot run scan-sources without a valid Konflux DB connection')
         runtime.konflux_db.bind(KonfluxBuildRecord)
 
-        self.github_token = os.getenv('GITHUB_TOKEN')
-        if not self.github_token:
+        github_token = os.getenv('GITHUB_TOKEN')
+        if not github_token:
             raise DoozerFatalError("GITHUB_TOKEN environment variable must be set")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)